warc_metadata_sidecar.py

```diff
@@ -193,6 +195,7 @@ def metadata_sidecar(archive_dir, warc_file, operator=None, publisher=None):
         text_mime = 0
         non_text = 0
         fido = ExtendFido()
+        warc_digest = ''
```

Comment:
Though from what I can tell this isn't likely to cause an actual problem, it seems odd to me to set `warc_digest` at the (W)ARC file level, when it is a record-level value. Perhaps instead it would make sense to set `warc_digest` to `None` inside the `else` block at line 227.